notebooks/official/training/multi_node_ddp_nccl_vertex_training_with_custom_container.ipynb

```diff
@@ -427,25 +368,30 @@
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "d48860504181"
+        "id": "2d242773d707"
       },
       "source": [
-        "### Import libraries and define constants"
+        "### Enable Artifact Registry API\n",
+        "You must enable the Artifact Registry API service for your project.\n",
```

Comment:
Recommend adding a link to the documentation:
- You must enable the [Artifact Registry API](https://cloud.google.com/artifact-registry/docs/reference/rest) service for your project